packages/project/test/merge/workflow-generator.ts

```diff
@@ -0,0 +1,67 @@
+import { randomUUID } from 'node:crypto';
```

Comment:
We need some unit tests around this.

It's also in the wrong place. I'm never sure where to put utility functions which are only used by unit tests.

Either it goes in `src/util/workflow-generator.ts`, or it goes in `test/util/workflow-generator.ts`, or `test/util/util.ts`